extensions/wikidata/src/org/openrefine/wikidata/editing/EditBatchProcessor.java

```diff
@@ -182,6 +183,14 @@ public void performEdit()
                 // Existing entities
                 EntityUpdate entityUpdate = update.toEntityUpdate(currentDocs.get(update.getEntityId().getId()));
                 editor.editEntityDocument(entityUpdate, false, summary, tags);
+                if (entityUpdate instanceof FullMediaInfoUpdate) {
```

Comment:
Our increased reliance on `instanceof` checks here is a sign that we need a better architecture for this… suggestions welcome :)